editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1308,6 +1309,36 @@ void SpriteFramesEditor::_frame_list_gui_input(const Ref<InputEvent> &p_event) {
 			_zoom_out();
 			// Don't scroll down after zooming out.
 			accept_event();
+		} else if (mb.is_valid() && mb->is_pressed() && mb->get_button_index() == MouseButton::RIGHT) {
+			Point2 pos = mb->get_position();
+			right_clicked_frame = frame_list->get_item_at_position(pos, true);
+			if (right_clicked_frame != -1) {
+				if (!menu) {
+					menu = memnew(PopupMenu);
+					add_child(menu);
+					menu->connect(SceneStringName(id_pressed), callable_mp(this, &SpriteFramesEditor::_menu_selected));
+				}
+				menu->clear();
+				menu->add_item(TTR("Show in FileSystem"));
+
+				menu->set_position(get_screen_position() + get_local_mouse_position());
+				menu->popup();
+			}
+		}
+	}
+}
+
+void SpriteFramesEditor::_menu_selected(int p_index) {
+	switch (p_index) {
+		case MENU_SHOW_IN_FILESYSTEM: {
```

Comment:
This `case` needs a `break;` after the `}`.

Reply:
done